packages/opentelemetry-metrics/src/MeterRegistry.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConsoleLogger } from '@opentelemetry/core';
+import * as types from '@opentelemetry/types';
```

Comment:
you are importing types twice

Reply:
good catch, also removed other duplicated types import in the update. Thanks. 😀